docs/source/kinds_of_types.rst

```diff
@@ -415,7 +415,7 @@ Disabling strict optional checking
 Mypy also has an option to treat ``None`` as a valid value for every
 type (in case you know Java, it's useful to think of it as similar to
 the Java ``null``). In this mode ``None`` is also valid for primitive
-types such as ``int`` and ``float``, and ``Optional[...]`` types are
+types such as ``int`` and ``float``, and :py:data:`~typing.Optional` types are
```

Comment:
Is it a similar case to [this one](https://github.com/python/mypy/pull/7652#discussion_r332748249)?